eng/pipelines/templates/steps/code-coverage.yml

```diff
@@ -0,0 +1,22 @@
+steps:
+  - task: Maven@3
+    displayName: 'Generate aggregate code coverage report'
+    condition: eq(variables['Coverage'], 'true')
+    inputs:
+      mavenPomFile: sdk/${{ parameters.ServiceDirectory }}/pom.xml
+      options: -Pcoverage
+      mavenOptions: '$(MemoryOptions) $(LoggingOptions)'
+      javaHomeOption: 'JDKVersion'
+      jdkVersionOption: $(JavaTestVersion)
+      jdkArchitectureOption: 'x64'
+      goals: jacoco:report-aggregate
```

Comment:
I'm not sure about this one. What happens if the corresponding pom.xml doesn't have "modules" section? Will jacoco:report-aggregate still work?

Reply:
Plus, Jacoco is plugged into "verify" already, there is no need to run this separately if "verify" was run already

Reply:
I don't think this is a change to existing behavior, just the template pieces being moved.